cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -11,21 +11,23 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: heapster-v1.1.0-beta1
+  name: heapster-v1.1.0.beta1
   namespace: kube-system
   labels:
     k8s-app: heapster
     kubernetes.io/cluster-service: "true"
+    version: v1.1.0.beta1
 spec:
   replicas: 1
   selector:
     matchLabels:
       k8s-app: heapster
+      version: v1.1.0.beta1
   template:
     metadata:
       labels:
         k8s-app: heapster
-        kubernetes.io/cluster-service: "true"
```

Comment:
With this label removed from the pods and replica sets, what will delete them when the version is upgraded in the future?